utils/native-cache/src/helpers.rs

```diff
@@ -0,0 +1,29 @@
+use std::{ffi::OsStr, path::Path};
+
+#[macro_export]
+macro_rules! log {
+    ($($tokens: tt)*) => {
+        if $crate::ENABLE_LOGS {
+            println!("cargo:warning={}", format!($($tokens)*))
+        }
+    }
+}
+
+/// Identify if the current path is a static or dynamic lib
+pub fn is_dyn_or_static_lib(p: impl AsRef<Path>) -> bool {
+    let p = p.as_ref();
+    if let Some((name, ext)) = p
+        .file_name()
+        .and_then(OsStr::to_str)
+        .and_then(|n| p.extension().map(|e| (n, e)))
+    {
+        (ext == "a" || ext == "so") && name.starts_with("lib")
```

Comment:
Not that it matters much as we are not using shared libraries, but still:
```suggestion
        (ext == "a" || ext == "so" || ext == "dylib") && name.starts_with("lib")
```

Reply:
THX